src/backend/distributed/sql/downgrades/citus--12.2-1--12.1-1.sql

```diff
@@ -17,6 +17,7 @@ DROP FUNCTION citus_internal.execute_command_on_remote_nodes_as_user(
 DROP FUNCTION citus_internal.mark_object_distributed(
     classId Oid, objectName text, objectId Oid, connectionUser text
 );
+DROP FUNCTION pg_catalog.citus_unmark_object_distributed(oid,oid,int,boolean);
```

Comment:
```suggestion

DROP FUNCTION pg_catalog.citus_unmark_object_distributed(oid, oid, int, boolean);
#include "../udfs/citus_unmark_object_distributed/10.0-1.sql"
```

Reply:
We shouldn't drop the older version of citus_unmark_object_distributed since it is being called without checkobjectexistence flag